src/backend/distributed/sql/udfs/worker_remove_jobdir/latest.sql

```diff
@@ -0,0 +1,6 @@
+CREATE FUNCTION pg_catalog.worker_remove_jobdir(bigint)
+    RETURNS void
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$worker_remove_jobdir$$;
+COMMENT ON FUNCTION pg_catalog.worker_remove_jobdir(bigint)
+    IS 'remove job in remote node';
```

Comment:
since we're running these as superuser, we can revoke access from all other users